components/frontend/package.json

```diff
@@ -8,14 +8,15 @@
     "fomantic-ui-css": "^2.9.2",
     "history": "^5.3.0",
     "react": "^18.2.0",
+    "react-datepicker": "^4.16.0",
```

Comment:
Does this also solve a lot of other gripes we've had with the datepicker?

Reply:
This one has proper keyboard navigation and seems to be actively maintained. Only disadvantage is that the calendar popup has no dark mode, but since it's a bit smaller than the previous one it is not too bad. If necessary we can always tinker with the CSS later on.